packages/clay-css/src/content/test_card_view_template.html

```diff
@@ -468,10 +468,10 @@ <h4 class="component-title">
 	<div class="container-fluid container-fluid-max-xl container-view sidenav-content">
 		<form>
 			<ul class="card-page">
-				<li class="card-page-item-header">
+				<li class="card-page-item card-page-item-header">
 					<h1 class="card-section-header">Group 1</h1>
 				</li>
-				<li class="card-page-item-directory">
+				<li class="card-page-item card-page-item-directory">
```

Comment:
hey @pat270 we have visual difference between the `.card-page-item-directory` and `.card-page-item-asset` classes?

Reply:
@matuzalemsteles no visual difference at this time. It's there just in case the Lexicon Team decides to deviate the layout for `card-type-asset` from `card-type-directory`. If it's easier to manage we can do something like `card-page-item` on the `li` and add a column class on `card-page card-page-4-col`, `card-page card-page-5-col`, etc. I will need to add these though.

Reply:
Thanks @pat270, for a moment we are always adding `.card-page-item.card-page-item-directory` we can keep this for now and when we have any changes on this, we will change.